apps/meteor/client/views/admin/apps/AppDetailsPage/AppDetailsPageHeader.tsx

```diff
@@ -36,16 +49,41 @@ const AppDetailsPageHeader = ({ app }: { app: App }): ReactElement => {
 					</Box>
 					{bundledIn && Boolean(bundledIn.length) && <BundleChips bundledIn={bundledIn} />}
 				</Box>
-				{app?.shortDescription && <Box mbe='x16'>{app.shortDescription}</Box>}
+
+				{shortDescription && (
+					<Box fontScale='p1' color='default' mbe='x16'>
+						{shortDescription}
+					</Box>
+				)}
 
 				<Box display='flex' flexDirection='row' alignItems='center' mbe='x16'>
 					<AppStatus app={app} installed={installed} isAppDetailsPage />
 					{(installed || isSubscribed) && <AppMenu app={app} isAppDetailsPage mis='x8' />}
 				</Box>
 				<Box display='flex' flexDirection='row' color='hint' alignItems='center'>
-					<Box fontScale='p2m' mie='x16'>
-						{t('By_author', { author: author?.name })}
+					<Box fontScale='c1' mie='x16'>
+						{author?.name}
+					</Box>
+					<Box fontScale='c1' color='stroke-light'>
+						|
 					</Box>
+					<Box fontScale='c1' mi='x16'>
+						{t('Version_version', { version })}
+					</Box>
+
+					{lastUpdated && (
+						<>
+							<Box fontScale='c1' color='stroke-light'>
```

Comment:
The `color` prop is for font color, you shouldn't pass a stroke token to it, only font colors :)

Reply:
Solved